src/wallet/transaction.h

```diff
@@ -140,8 +134,7 @@ class CWalletTx
     mutable CAmount nChangeCached;
 
     CWalletTx(const CWallet* wallet, CTransactionRef arg)
```

Comment:
7bc16a0b421380a1682ca9d97b7f7fc140fb1c5e

nit, `CWalletTx(const CWallet*, CTransactionRef arg)` to make it clear wallet is unused.

Reply:
re: https://github.com/bitcoin/bitcoin/pull/21206#discussion_r579802970

> nit, `CWalletTx(const CWallet*, CTransactionRef arg)` to make it clear wallet is unused.

Good catch! Removed unused argument in this commit instead of later commit